app/controllers/oauth_controller.rb

```diff
@@ -129,7 +129,7 @@ def is_user_authorized
       s3 = Aws::S3::Client.new({
         access_key_id: ENV['AWS_KEY_ID'],
         secret_access_key: ENV['AWS_SECRET'],
-        region: 'us-east-1'
+        region: ENV['SQS_REGION']
```

Comment:
We might want to consider rename this env variable, if we use it more than just for SQS.